src/opencl_bitcoin_fmt_plug.c

```diff
@@ -128,6 +128,7 @@ static cl_int cl_error;
 static cl_mem mem_in, mem_salt, mem_state, mem_cracked;
 static struct fmt_main *self;
 static cl_kernel init_kernel, final_kernel;
+static int init_kernel_max_lws;
```

Comment:
Nitpick: OpenCL spec and thus we use `size_t` for LWS elsewhere in this file, so maybe `init_kernel_max_lws` would be cleaner as a `size_t` rather than an `int`.

Reply:
True. In this case it really doesn't matter but I'll change it for good measure.